libpromises/cf3parse.y

```diff
@@ -749,16 +749,7 @@ litem:                 IDENTIFIER
 
                      | QUOTED_STRING
                        {
-                           ParserDebug("\tP:%s:%s:%s:%s list append: "
-                                       "qstring = %s\n",
-                                       ParserBlockString(P.block), P.blocktype, P.blockid,
-                                       (P.currentclasses ?
-                                            P.currentclasses : "any"),
-                                       P.currentstring);
-                           RlistAppendScalar((Rlist **) &P.currentRlist,
-                                             (void *) P.currentstring);
-                           free(P.currentstring);
-                           P.currentstring = NULL;
+                           ParserHandleQuotedListItems();
```

Comment:
Not a big deal, but something to consider for the next time:

If you copied this code into a function without changing anything, only whitespace, in a separate commit, it would be easier to review and see what changes you made to it.